CONTRIBUTING.md

```diff
@@ -210,4 +223,4 @@ nothing.
 #### Step 7: Deploy updates to the `master` branch
 If the `master` branch has been updated, you can push all the changes
 to the dash-bio remote. However, make sure that the `Procfile` on the
-dash-bio remote stays the same.
+dash-bio repository stays the same.
```

Comment:
In any case (whether one is working on a fork or a clone), there's always only one repository and it's the `dash-bio` repository. We are always in the "dash-bio repository" so clearly the above doesn't reflect what its author intended... We are always pushing to and pulling from the "dash-bio repository." What we are paying attention to here are *branches* and *remotes*. Please reword the above with the intended information.

Reply:
Resolved by PR #188